mmcls/models/backbones/mlp_mixer.py

```diff
@@ -0,0 +1,140 @@
+import torch.nn as nn
+from mmcv.cnn import build_activation_layer, build_norm_layer
+from mmcv.cnn.bricks.drop import build_dropout
+from mmcv.runner.base_module import BaseModule, Sequential
+from mmcv.utils.misc import to_2tuple
+
+from mmcls.models.utils import PatchEmbed
+from ..builder import BACKBONES
+
+
+class Mlp(BaseModule):
```

Comment:
@mzr1996  Should we consider to add this class into `mmcv` or make it as a registered module. Passing `act_layer_cfg` multiple times is dumb.

Some existing modules like `StackedLinearClsHead` in `mmcls` and `FFN` in `mmcv` does similar thing. However they do not match the requirement here. The parameter `num_classes` in `StackedLinearClsHead`  is misleading, `FFN` requires the input dimension and output dimension to be the same.

Reply:
If this module is a simple forward network without many other configs, just using a `Sequence` in `MixerBlock` is enough.

Reply:
And `StackedLinearClsHead` is a head, it shouldn't be used as a module in the backbone.